packages/blockchain/src/types.ts

```diff
@@ -82,6 +84,14 @@ export interface BlockchainOptions {
    */
   validateConsensus?: boolean
 
+  /**
+   * Options to pass in string to define consensus
+   * Or to pass in custom consensusAlgorithm that correctly implements `Consensus`
+   */
+
+  consensus?: 'casper' | 'clique' | 'ethash'
```

Comment:
```suggestion
  consensus?: ConsensusAlgorithm
```

Let's use the `ConsensusAlgorithm` enum from `common` here